.circleci/config.yml

```diff
@@ -124,7 +124,7 @@ jobs:
       - image: circleci/ruby:2.4
     working_directory: /tmp/deploy
     environment:
-      - DISTROS: "trusty xenial bionic el6 el7"
+      - DISTROS: "trusty xenial bionic el7 el8"
```

Comment:
You don't want to replace `el6` with `el8`. Just adding `el8` should be good with more `parallelism` to CircleCI config.

Reply:
Yes I have a paid plan and I’ve spent like $30 in circleci credit so I was trying to minimize the number of containers building

Reply:
Will revert.